recipes/r-roi.plugin.quadprog/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set version = '1.0-1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-roi.plugin.quadprog
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ROI.plugin.quadprog_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ROI.plugin.quadprog/ROI.plugin.quadprog_{{ version }}.tar.gz
+  sha256: e891778a451cb8688fcb885e02d4ac51c09c83f8fb5344a4c271cad469712087
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-roi >=0.3_0
+    - r-quadprog
+    - r-slam
+  run:
+    - r-base
+    - r-roi >=0.3_0
+    - r-quadprog
+    - r-slam
+
+test:
+  commands:
+    - $R -e "library('ROI.plugin.quadprog')"           # [not win]
+    - "\"%R%\" -e \"library('ROI.plugin.quadprog')\""  # [win]
+
+about:
+  home: http://roi.r-forge.r-project.org/, https://r-forge.r-project.org/projects/roi/
+  license: GPL-3.0-or-later
```

Only see `GPL-3`, so

```suggestion
  license: GPL-3.0-only
```